### PR TITLE
ensure timeout error is shown on u2f timeout

### PR DESCRIPTION
**Pull request by zeripath**
U2F timeout error is not currently shown due to miss match the '-' and '_' in the class names on u2f_error.tmpl

### CODE REVIEW

File: web_src/js/index.js [hunk 1]
```diff
@@ -2243,7 +2243,7 @@ function u2fError(errorType) {
     2: $('#u2f-error-2'),
     3: $('#u2f-error-3'),
     4: $('#u2f-error-4'),
-    5: $('.u2f-error-5')
+    5: $('.u2f_error_5')
```
Comment (6543) [inline web_src/js/index.js:2246]: is this inconsistency really right?

Comment (zeripath) [reply web_src/js/index.js:2246]: yes look at line 29 in u2f_error.tmpl

**Comment by zeripath** [comment]
I would argue we should rename these classes and the locale keys to match:

interface ErrorCode {
    const short OK = 0;
    const short OTHER_ERROR = 1;
    const short BAD_REQUEST = 2;
    const short CONFIGURATION_UNSUPPORTED = 3;
    const short DEVICE_INELIGIBLE = 4;
    const short TIMEOUT = 5;
};

from https://fidoalliance.org/specs/fido-u2f-v1.0-nfc-bt-amendment-20150514/fido-u2f-javascript-api.html

But that would not be backportable.

**Review by 6543 — APPROVED** [review]

File: web_src/js/index.js (hunk 1 above)
Comment (techknowlogick) [reply web_src/js/index.js:2246]: wouldn't it be better to change L29 of the template to `u2f-error-5` instead to retain consistency?

**Comment by techknowlogick** [comment]
mmm... yes, that would be much better. although as you said, not backportable. Lets save the refactor for another PR then. I'll create an issue for the refactor and tag it as "good-first-issue" in case someone is looking for a PR to ease their way into contributing to Gitea.

**Review by techknowlogick — APPROVED** [review]
approving per my latest comment.

**Comment by 6543** [comment]
:rocket:

**Pull request merged**